content/smart.md

```diff
@@ -0,0 +1,140 @@
+
+### SMART Without FHIR ###
+
+Technically speaking, the SMART specification requires that a FHIR server exist so that a SMART app can communicate with the FHIR server when the app opens in order to determine the location of the authorization (OAuth 2) server. However, outside of this requirement, FHIR does not need to be leveraged otherwise by the SMART app.
+
+## Technical ##
+
+### Screen Resolution ###
+
+The recommended screen resolution varies based on the available screen's real estate. Instead of targeting a specific resolution, the app should be designed to be responsive, and the app should adjust according to the available screen sizes.
+
+### Browser Requirement ###
+
+Apps should be designed to work in any browser for best user experience. For provider facing apps that run on the *Cerner Millennium* platform, Cerner currently supports only the embedded Internet Explorer browser (IE) at the moment. The minimum-supported embedded browser is IE 10. The latest supported browser varies based on the version of the browser that is currently installed on the device.
+
+### HTML5 DOCTYPE ###
+
+The doctype declaration tells the browser what version of HTML the document is written in. It is a good idea to include this declaration so that the browser will render the document correctly. It must be included at the very first line of the document, right above the `<html>` tag. 
+
+```<!DOCTYPE html>```
+
+### X-UA-Compatible Tag ###
+
+In order for the embedded IE browser to render the app in the latest supported IE version, set the following HTML tag in the first line of the `<head>` section for all pages:
+
+```<meta http-equiv="X-UA-Compatible" content="IE=edge">```
+
+This is required because the default embedded IE version is set to another version for compatibility with other solutions.
+
+
+### Technology and Framework for SMART Apps ###
+
+What technology and framework you use is really up to you and your company. Regardless of which technology, framework, or language you choose, make sure that your app is fast, responsive, and adaptable to various screen sizes.  We recently created [ASCVD Risk SMART app][ASCVD-RISK-APP] using React.js.
+
+### Open Source FHIR Client Libraries ###
+
+To start development quickly, there is an open source [fhir-client JavaScript library][FHIR-CLIENT-JS] that takes care of the OAuth2 handshake and provides a built-in library to call FHIR resources. The library is usable but has several known issues in previous versions. One particular issue is around the usage of the `sessionStorage` property. Cerner highly recommends that you upgrade the library to version [v0.1.10][FHIR-CLIENT-v0.1.10] or higher to correct a known patient safety issue. You also need to download and include this [additional code][FHIR-CLIENT-CERNER-ADDITIONS] into your project to correct this issue.
```

Comment:
We don't highly recommend, we require this. We need to be much stronger on this.